packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -893,36 +893,40 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
    * Updates the agent device type.
    * This method allows the agent to change their device type (e.g., from BROWSER to EXTENSION or anything else).
    * It will also throw an error if the new device type is the same as the current one.
-   * @param data type is AgentDeviceUpdate - The data required to update the agent device type, including the new login option and dial number.
+   * @param data type is AgentProfileUpdate - The data required to update the agent device type, including the login option, team id and dial number.
    * @returns Promise<UpdateDeviceTypeResponse>
    * @throws Error
    * @example
    * ```typescript
    * const data = {
    *   loginOption: 'EXTENSION',
    *   dialNumber: '1234567890',
+   *   teamId: 'team-id-if-needed', // Optional, if not provided, current team ID will be used
    * };
-   * const result = await webex.cc.updateAgentDeviceType(data);
+   * const result = await webex.cc.updateAgentProfile(data);
    * ```
    */
-  public async updateAgentDeviceType(data: AgentDeviceUpdate): Promise<UpdateDeviceTypeResponse> {
+  public async updateAgentProfile(data: AgentProfileUpdate): Promise<UpdateDeviceTypeResponse> {
     this.metricsManager.timeEvent([
       METRIC_EVENT_NAMES.AGENT_DEVICE_TYPE_UPDATE_SUCCESS,
       METRIC_EVENT_NAMES.AGENT_DEVICE_TYPE_UPDATE_FAILED,
     ]);
 
     const trackingId = `WX_CC_SDK_${uuidv4()}`;
 
-    LoggerProxy.info(`[${trackingId}] updateAgentDeviceType | starting profile update`, {
+    LoggerProxy.info(`[${trackingId}] updateAgentProfile | starting profile update`, {
       module: CC_FILE,
-      method: this.updateAgentDeviceType.name,
+      method: this.updateAgentProfile.name,
```

Comment:
Didn't we check that in minified versions this name wasn't going through and we had to mention strings here?

cc - @rsarika